scripts/setup-environment.sh

```diff
@@ -143,6 +147,10 @@ echo "PN_FRIENDS_SECURE_SERVER_HOST=$server_ip" >>./friends.local.env
 echo "PN_WIIU_CHAT_SECURE_SERVER_LOCATION=$server_ip" >>./wiiu-chat.local.env
 echo "PN_SMM_SECURE_SERVER_HOST=$server_ip" >>./super-mario-maker.local.env
 echo "PN_SPLATOON_SECURE_SERVER_HOST=$server_ip" >>./splatoon.local.env
+echo "PN_MINECRAFT_SECURE_SERVER_HOST=$server_ip" >>./minecraft-wiiu.local.env
+
+echo "PN_MINECRAFT_ALLOW_PUBLIC_MATCHMAKING=0" >>./minecraft-wiiu.local.env
+echo "# 0 = disable, 1 = enable" >>./minecraft-wiiu.local.env
```

Comment:
A persistent setting like this should go in `minecraft.env` because otherwise, it will be reset every time someone re-runs the environment setup script (which happens whenever the setup script is run, including on updates). I agree that this should be set to 0 by default for safety reasons. The extra comment (`# 0 = disable, 1 = enable`) isn't necessary, as it's pretty obvious.